public/app/features/alerting/unified/components/admin/AlertmanagerConfigSelector.tsx

```diff
@@ -0,0 +1,87 @@
+import { css } from '@emotion/css';
+import React, { useMemo } from 'react';
+
+import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import { Select, useStyles2 } from '@grafana/ui';
+import { AlertManagerCortexConfig } from 'app/plugins/datasource/alertmanager/types';
+
+import { alertmanagerApi } from '../../api/alertmanagerApi';
+import { GRAFANA_RULES_SOURCE_NAME } from '../../utils/datasource';
+
+import { FormValues } from './AlertmanagerConfig';
+import { ConfigEditor } from './ConfigEditor';
+
+export interface ValidAmConfigOption {
+  label?: string;
+  value?: AlertManagerCortexConfig;
+}
+
+interface AlertmanagerConfigSelectorProps {
+  onChange: (selectedOption: ValidAmConfigOption) => void;
+  selectedAmConfig?: ValidAmConfigOption;
+  defaultValues: FormValues;
+  onSubmit: (values: FormValues, fetchLatestConfig: boolean, oldConfig?: AlertManagerCortexConfig) => void;
+  readOnly: boolean;
+  loading: boolean;
+}
+
+export default function AlertmanagerConfigSelector({
+  onChange,
+  selectedAmConfig,
+  defaultValues,
+  onSubmit,
+  readOnly,
+  loading,
+}: AlertmanagerConfigSelectorProps): JSX.Element {
+  const { useGetValidAlertManagersConfigQuery } = alertmanagerApi;
+
+  const styles = useStyles2(getStyles);
+
+  const { currentData: validAmConfigs, isLoading: isFetchingValidAmConfigs } = useGetValidAlertManagersConfigQuery();
+
+  const validAmConfigsOptions = useMemo(() => {
+    if (!validAmConfigs?.length) {
+      return [];
+    }
+
+    const configs: ValidAmConfigOption[] = validAmConfigs.map((config, index) => ({
+      label: `Config ${index + 1}`,
+      value: config,
+    }));
+    onChange(configs[0]);
+    return configs;
+  }, [validAmConfigs, onChange]);
+
+  return (
+    <>
+      {!isFetchingValidAmConfigs && validAmConfigs && validAmConfigs.length > 0 ? (
```

Comment:
```suggestion
      {!isFetchingValidAmConfigs && validAmConfigs?.length > 0 ? (
```

Reply:
I tried to do that but I get this error 😕 
![image](https://user-images.githubusercontent.com/6271380/208120677-aa23a3f2-85ae-4a60-8b2a-6fd294405703.png)

Reply:
```suggestion
      {!isFetchingValidAmConfigs && Boolean(validAmConfigs?.length > 0 )? (
```